ckan/lib/webassets_tools.py

```diff
@@ -135,12 +161,20 @@ def _to_tag(url: str, type_: str):
 
 
 def render_assets(type_: str) -> Markup:
+    """ Renders the webassets of a specific type.
+
+    This function is called at the end of the base.html template. It
+    will iterate trough g.webassets to return the necessary html tags to
+    include the assets in the html response.
+    """
     try:
         assets = g.webassets
     except AttributeError:
+        logger.warn("Webassets are not initialized.")
         return Markup(u'')
 
     if not assets:
+        logger.warn("Webassets are initialized but none was found.")
```

Comment:
I added a warning message since CKAN will not render properly if webassets are not initialized when this function gets call.